src/devices/Ina226/Ina226DelayHelper.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+
+namespace Iot.Device.Ina226
+{
+    /// <summary>
+    /// Helpers for short waits.
+    /// </summary>
+    internal static class Ina226DelayHelper
+    {
+        /* GetTimestamp() currently can take ~300ns. We hope to improve this to get better
+         * fidelity for very tight spins.
+         *
+         * SpinWait currently spins to approximately 1μs before it will yield the thread.
+         */
+
+        private const long TicksPerSecond = TimeSpan.TicksPerSecond;
+        private const long TicksPerMillisecond = TimeSpan.TicksPerMillisecond;
+        private const long TicksPerMicrosecond = TimeSpan.TicksPerMillisecond / 1000;
+
+        /// <summary>A scale that normalizes the hardware ticks to <see cref="TimeSpan" /> ticks which are 100ns in length.</summary>
+        private static readonly double s_tickFrequency = (double)TicksPerSecond / Stopwatch.Frequency;
+
+        /// <summary>
+        /// Delay for at least the specified <paramref name="time" />.
+        /// </summary>
+        /// <param name="time">The amount of time to delay.</param>
+        /// <param name="allowThreadYield">
+        /// True to allow yielding the thread. If this is set to false, on single-proc systems
+        /// this will prevent all other code from running.
+        /// </param>
+        public static void Delay(TimeSpan time, bool allowThreadYield)
+        {
+            long start = Stopwatch.GetTimestamp();
```

Comment:
any reason you cannot use: https://github.com/dotnet/iot/blob/main/src/devices/Common/System/Device/DelayHelper.cs#L35 ?

Reply:
same for methods below, you should be able to include this file similarly as we do in other projects

Reply:
on Windows you can search for references: `findstr /sinpc:"DelayHelper.cs" *.csproj` from the root directory